Sources/Swift/Transactions/SentrySpanKey.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+@objcMembers @objc(SentrySpanKey)
+class SentrySpanKey: NSObject {
+    /// Used to set the number of bytes processed in a file span operation
+    static let fileSize = "file.size"
+
+    /// Used to set the path of the file in a file span operation
+    static let filePath = "file.path"
```

Comment:
Are these meant to be included in this PR?

Reply:
Yes, I did not think it has a lot of value in splitting up another PR